Docker/Dockerfile.O3DE

```diff
@@ -0,0 +1,166 @@
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+
+ARG ROS_VERSION=humble
+ARG UBUNTU_VERSION=jammy
+
+FROM ros:${ROS_VERSION}-ros-base-${UBUNTU_VERSION}
+
+ENV WORKSPACE=/data/workspace
+
+WORKDIR $WORKSPACE
+
+# O3DE Arguments
+ARG O3DE_REPO=https://github.com/o3de/o3de
+ARG O3DE_BRANCH=development
```

Comment:
Shouldn't we keep the default configuration linked with the `main` releases? Or even the tag that was tested?

Reply:
I generally like to keep the same tag as the branch that the docker is in,  and in this case its the development branch of the demo. If this was merged into main, then the default branch tags here should do the same.